masu/database/report_db_accessor.py

```diff
@@ -232,7 +232,7 @@ def update_summary_tables(self, provider, start_date, end_date, manifest_id=None
             None
 
         """
-        if provider != 'AWS':
+        if provider not in ('AWS', 'AWS-local'):
```

Comment:
We'll just have to watch who merges first, as my change will invalidate this. My vote is merge yours first, and then I can clean mine.